patient_portal/import_template_ui.php

```diff
@@ -13,52 +13,59 @@
  * @author Jerry Padgett <sjpadgett@gmail.com>
  * @link http://librehealth.io
  *
- * Please help the overall project by sending changes you make to the authors and to the LibreEHR community.
+ * Please help the overall project by sending changes you make to the authors and to the LibreHealth EHR community.
  *
  */
 $sanitize_all_escapes=true;
 $fake_register_globals=false;
 require_once("../interface/globals.php");
 $getdir = isset($_POST['sel_pt']) ? $_POST['sel_pt'] : 0;
 if( $getdir > 0){
-    $tdir = $GLOBALS['OE_SITE_DIR'] .  '/onsite_portal_documents/templates/' . $getdir . '/';
+    $tdir = $GLOBALS['OE_SITE_DIR'] .  '/documents/onsite_portal_documents/templates/' . $getdir . '/';
```

Comment:
onsite_ is a redacted usage.  I think that patient_portal_documents is a lot more descriptive.

Reply:
Kiosk will have a directory here also